Components/navigation/navbar.css

```diff
@@ -0,0 +1,59 @@
+.nav-bar {
+  width: 100%;
+  height: 5rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 1rem;
+  z-index: 1;
+}
+
+.boxShadow {
+  box-shadow: 0px 2px 10px 4px rgb(0 0 0 / 5%);
+}
+
+.nav-innerContainer {
+  display: flex;
+  justify-content: space-around;
+  width: 16%;
+  color: var(--light-clr);
+}
+
+.nav-icon-container {
+  width: 10%;
+}
+
+.nav-heading {
+  display: inline;
+  margin: 0;
+}
+
+.text-content {
+  padding: 1rem;
+  padding-top: 4rem;
+}
+
+.nav-icon {
+  font-size: 1.4rem;
+}
+
+.docSite-nav-clr {
+  color: var(--font-clr);
+}
+
+.nav_searchBar {
+  outline: none;
+  width: 40rem;
+  padding: 0.5rem;
+  border: 1px solid var(--gray-shade5);
+}
+
+.searchBar_icon {
+  height: auto;
+  display: flex;
```

Comment:
you can create a separate flex class for reusability.

Reply:
Actually I have created d-flex class, but forgot to use it , done with the changes , thanks for the review